src/remix/sdk-setup.ts

```diff
@@ -36,6 +38,52 @@ export type PartialRemixConfig = {
 };
 
 const REMIX_CONFIG_FILE = 'remix.config.js';
+const REMIX_REVEAL_COMMAND = 'npx remix reveal';
+
+export function runRemixReveal(isTS: boolean): void {
+  // Check if entry files already exist
+  const clientEntryFilename = `entry.client.${isTS ? 'tsx' : 'jsx'}`;
+  const serverEntryFilename = `entry.server.${isTS ? 'tsx' : 'jsx'}`;
+
+  const clientEntryPath = path.join(process.cwd(), 'app', clientEntryFilename);
+  const serverEntryPath = path.join(process.cwd(), 'app', serverEntryFilename);
+
+  if (fs.existsSync(clientEntryPath) && fs.existsSync(serverEntryPath)) {
+    clack.log.info(
+      `Found entry files ${chalk.cyan(clientEntryFilename)} and ${chalk.cyan(
+        serverEntryFilename,
+      )}.`,
+    );
+  } else {
+    clack.log.info(
+      `Couldn't find entry files in your project. Trying to run ${chalk.cyan(
+        REMIX_REVEAL_COMMAND,
+      )}...`,
+    );
+
+    childProcess.execSync(REMIX_REVEAL_COMMAND, {
+      stdio: 'inherit',
```

Comment:
l: Can we pipe the output from the command and wrap it in a `clack` call? Currently this somewhat breaks the UI:
![image](https://github.com/getsentry/sentry-wizard/assets/8420481/4aaf2f93-d5ff-4252-8c5f-fdae19225a38)

Reply:
Updated 👍 -> [a184b9f](https://github.com/getsentry/sentry-wizard/pull/495/commits/a184b9f1915ef27ef8a309d69d3afeb39c8f5101)